tests/test_organizations/test_organizations_boto3.py

```diff
@@ -1025,14 +1092,23 @@ def test_list_tags_for_resource_errors():
     client.create_organization(FeatureSet="ALL")
 
     with pytest.raises(ClientError) as e:
-        client.list_tags_for_resource(ResourceId="000000000000")
+        client.list_tags_for_resource(ResourceId="000x00000A00")
     ex = e.value
     ex.operation_name.should.equal("ListTagsForResource")
     ex.response["ResponseMetadata"]["HTTPStatusCode"].should.equal(400)
     ex.response["Error"]["Code"].should.contain("InvalidInputException")
     ex.response["Error"]["Message"].should.equal(
         "You provided a value that does not match the required pattern."
     )
+    with pytest.raises(ClientError) as e:
```

Comment:
Looks like this test is duplicated three times

Reply:
the test is done with 3 different resources,

one test tests tagging an account and checks the tags
second test tests tagging an root and checks the tags
etc.

I can consolidate the tests and do a for loop with the different resource types. or consolidate parts in a helper function. What is your preferred approach?

Reply:
My bad, I didn't mean the entire test - just the call to `client.list_tags_for_resource(ResourceId="000000000000")`. 
That method-call, and the assertion that it throws a TargetNotFoundException, is duplicated.

Reply:
You are right, I was also not paying attention.  This should check 3 different functions, list_tags, create_tag etc. will update!

Reply:
I fixed the tests. They now all test different responses per account type